krun/results.py

```diff
@@ -11,7 +11,14 @@ class Results(object):
     Can be serialised to disk.
     """
 
+    # We use this to detect times where a results instance is loaded prior to a
+    # process execution. This wouuld be bad, as the results can be big and
+    # cause memory to fragment.
+    OK_TO_INSTANTIATE = False
```

Comment:
By making this all capitals, it suggests it's a constant -- which it isn't. I think we should make it `ok_to_instantitate` but make it clear that it's a class variable.